### PR TITLE
Project create success page update

### PR DESCRIPTION
**Pull request by johnnyd-eth**
Fixes JB-324 (implements https://www.figma.com/file/UE4g6PFX3d7odydhzAeD5j/JBM-Website-Updates-March-2023?node-id=1758%3A203088&t=XKSuX1ko0UxhCSN6-1)

Screen Shot 2023-05-04 at 9 09 56 pm

Screen Shot 2023-05-04 at 9 09 46 pm

### CODE REVIEW

File: src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx [hunk 1]
```diff
@@ -48,28 +49,50 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
...
+        {deployGreeting}
+      </div>
+      <XLButton
+        type="primary"
+        onClick={handleGoToProject}
+        loading={gotoProjectClicked}
+        className="mt-8"
+      >
+        <span>
+          <div className="flex items-center gap-3 text-white">
```
Comment (tomquirk) [inline src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx:70]: **issue** cant have div within span, there'll be a console error

File: src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx [hunk 2]
```diff
@@ -48,28 +49,50 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
...
-          type="primary"
-          onClick={handleGoToProject}
-          loading={gotoProjectClicked}
-        >
-          <span>
-            <Trans>Go to project</Trans>
-          </span>
-        </Button>
+      <Image
+        src="/assets/images/create-success-hero.webp"
```
Comment (tomquirk) [inline src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx:53]: **suggestion** if you import this image using a JS import (`import image from "/public/assets..."`), you don't need to define width/height (something i learned recently!)

**Review by tomquirk — CHANGES_REQUESTED** [review]
@johnnyd-eth minor comments!

File: src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx (hunk 2 above)
Comment (johnnyd-eth) [reply src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx:53]: Done but is there any benefit in not using height and width props to downsize?

File: src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx [hunk 3]
```diff
@@ -48,28 +51,44 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
...
+      <div className="mt-4 text-lg font-normal text-grey-600 dark:text-slate-200">
+        {deployGreeting}
+      </div>
+      <XLButton
+        type="primary"
+        onClick={handleGoToProject}
+        loading={gotoProjectClicked}
+        className="mt-8 flex items-center gap-3 text-white"
+      >
+        <span className="text-[16px]">
```
Comment (tomquirk) [inline src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx:67]: never need to use px in font sizes. `text-base` will work here

Comment (johnnyd-eth) [reply src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx:67]: Ah true that's right, forgot about `base`

File: src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx (hunk 1 above)
Comment (johnnyd-eth) [reply src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx]: Ah yep, fixed and still all works:

Screen Shot 2023-05-05 at 9 50 19 am

File: src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx (hunk 2 above)
Comment (tomquirk) [reply src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx]: Hows it look after running lighthouse?

Comment (johnnyd-eth) [reply src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx]: Actually not sure how to run lighthouse for something like this, where it's on the `/create` page but required a state change (project creation) to load. Is it possible?

Comment (tomquirk) [reply src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx]: hmm that is a great point.

**Pull request merged**